frontend/lint-staged.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  '*.ts': filenames => `npm run lint:fix  -- --files {${filenames.join(',')}}`,
```

Comment:
Oh damn... https://github.com/angular/angular-cli/issues/13078
I really don't why they're wrapping tslint and creating yet another tool with different quirkss.

Reply:
Yep, took 1h~1,5h of my life but it works now ;)